docs/source/features/passes/quant_onnx.md

```diff
@@ -122,3 +122,26 @@ Olive consolidates the Vitis™ AI quantization into a single pass called VitisA
 }
 ```
 Please refer to [VitisAIQuantization](vitis_ai_quantization) for more details about the pass and its config parameters.
+
+## NVIDIA TensorRT Model Optimizer-Windows
+Olive also integrates [TensorRT Model Optimizer-Windows](https://github.com/NVIDIA/TensorRT-Model-Optimizer)
+
+The TensorRT Model Optimizer-Windows is engineered to deliver advanced model compression techniques, including quantization, to Windows RTX PC systems. Specifically tailored to meet the needs of Windows users,it is optimized for rapid and efficient quantization, featuring local GPU calibration, reduced system and video memory consumption, and swift processing times.
+
+The primary objective of the TensorRT Model Optimizer-Windows is to generate optimized, standards-compliant ONNX-format models for DirectML backends. This makes it an ideal solution for seamless integration with ONNX Runtime (ORT) and DirectML (DML) frameworks, ensuring broad compatibility with any inference framework supporting the ONNX standard.
+
+Olive consolidates the NVIDIA TensorRT Model Optimizer-Windows quantization into a single pass called NVModelOptQuantization which supports AWQ algorithm.
+
+### Example Configuration
+```json
+"quantization": {
+    "type": "NVModelOptQuantization",
+    "algorithm": "AWQ",
+    "tokenizer_dir": "microsoft/Phi-3-mini-4k-instruct",
```

Comment:
not relevant to this PR but I wanted to get your thoughts on making tokenizer_dir optional in a future update. the model builder saves the tokenizer as part of the onnx model directory. We also have an option in torch export pass to save tokenizer and other files along with the exported model. 
In such cases, if `tokenizer_dir` is not provided we could try loading from the parent of `model.model_path` and only raise an error if it fails. This way it becomes more seamless and user doesn't have to remember to update the path in multiple places.